pom.xml

```diff
@@ -181,7 +182,13 @@
 
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>blueocean-plugin</artifactId>
```

Comment:
`blueocean-plugin` is only the name of the folder module, the artifact name is `blueocean`, see [pom.xml](https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-plugin/pom.xml#L11)